src/backend/src/app.module.ts

```diff
@@ -1,19 +1,40 @@
 import { Module } from '@nestjs/common';
 import { TypeOrmModule } from '@nestjs/typeorm';
 
+// import { APP_GUARD } from '@nestjs/core';
+import { KeycloakConnectModule } from 'nest-keycloak-connect';
 import { AppController } from './app.controller';
 import { AppService } from './app.service';
 import { configService } from './config/config.service';
 import { HealthModule } from './health/health.module';
 import { PpqModule } from './modules/ppq/ppq.module';
+import { HttpModule } from '@nestjs/axios';
+import { AuthModule } from './modules/auth/auth.module';
 
 @Module({
   imports: [
     TypeOrmModule.forRoot(configService.getTypeOrmConfig()),
     HealthModule,
+    AuthModule,
     PpqModule,
+    HttpModule.register({
+      timeout: 5000,
+      maxRedirects: 5,
+    }),
+    KeycloakConnectModule.register(configService.getKeycloakConfig()),
   ],
   controllers: [AppController],
-  providers: [AppService],
+  providers: [
+    AppService,
```

Comment:
Commented out code in a PR? I would at least add a comment to explain what this is.

Reply:
i will remove the commented code